src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipPdfType.php

```diff
@@ -51,6 +51,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'attr' => ['placeholder' => 'YYYY-MM-DD'],
                     'data' => $now,
                     'empty_data' => $now,
+                    'label' => $this->trans('From', 'Admin.Global'),
+                    'help' => $this->trans('Format: 2011-12-31 (inclusive).', 'Admin.Orderscustomers.Help'),
```

Comment:
```suggestion
                    'help' => $this->trans('Format: %s (inclusive).', 'Admin.Orderscustomers.Help', [date('Y-m-d')]),
```

We can use the same wording as here: https://github.com/PrestaShop/PrestaShop/pull/27605/files

Reply:
Done :heavy_check_mark: